### PR TITLE
Fix SetSlot (#1144)

### PR DESCRIPTION
**Pull request by ReinforceZwei**
#1144

### CODE REVIEW

File: MinecraftClient/McClient.cs [hunk 1]
```diff
@@ -1723,10 +1723,23 @@ public void OnSetSlot(byte inventoryID, short slotID, Item item)
...
-                    inventories[0].Items.Remove(-1);
+                if (inventories.ContainsKey(0))
+                {
+                    if (item != null)
+                        inventories[0].Items[-1] = item;
+                    else
+                        inventories[0].Items.Remove(-1);
+                }
+            }
+            else if (inventoryID == 254) // To add item to player inventory without animation
```
Comment (ORelio) [inline MinecraftClient/McClient.cs:1734]: To limit copy and paste, how about doing this at the beginning of function:
```C#
if (inventoryID == 254)
    inventoryID = 0;
```
Handling for inventory ID 254 is slightly different than other inventories, not sure if that's intentional.

Comment (ReinforceZwei) [reply MinecraftClient/McClient.cs]: Done.

Comment (ReinforceZwei) [reply MinecraftClient/McClient.cs]: > Handling for inventory ID 254 is slightly different than other inventories, not sure if that's intentional.

It should be the same except no item adding animation.

Comment (ORelio) [reply MinecraftClient/McClient.cs]: Ok, since we do not animate this... it's the same 😄

**Pull request merged**